packages/app/src/AuthPage.tsx

```diff
@@ -45,6 +48,23 @@ export default function AuthPage({ action }: { action: 'register' | 'login' }) {
     }
   }, [installation, isRegister, router]);
 
+  const [currentPassword, setCurrentPassword] = useState<string>('');
```

Comment:
sorry to pile on late on this PR... we're a bit inconsistent in the project but we're trying to move frontend forms to be more based off of `react-hook-form` when possible. 

In this case we're already using the hook on line 30 above here, and the hook allows for listening to values in the form via [`watch`](https://react-hook-form.com/docs/useform/watch) so you don't have to manage the state/updating yourself here. Instead you should be able to do something like:

`const currentPassword = watch('password');`

Reply:
100% love that, much nicer, commit coming momentarily